### PR TITLE
Migrate docs from wiki

### PR DESCRIPTION
**Pull request by diddy10**

### CODE REVIEW

**Comment by proxxz** [comment]
@voorth PR is ready for review.

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.399</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>2.37</version><!-- which version of Jenkins is this plugin built against? -->
```
Comment (bromps) [inline pom.xml:6]: I don't quite remember correctly, but I think this change was needed to get the build to pass.

Comment (oleg-nenashev) [reply pom.xml:6]:
```suggestion
    <version>4.12</version>
```

For the latest stuff. 2.37 is pretty old

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[[built-on-column-plugin]]
+= Built On Column Plugin
+
+link:https://ci.jenkins.io/job/Plugins/job/built-on-column-plugin/job/master/[image:https://ci.jenkins.io/job/Plugins/job/built-on-column-plugin/job/master/badge/icon[Build]]
+link:https://github.com/jenkinsci/built-on-column-plugin/graphs/contributors[image:https://img.shields.io/github/contributors/jenkinsci/built-on-column-plugin.svg?color=blue[Contributors]]
+link:https://plugins.jenkins.io/built-on-column/[image:https://img.shields.io/jenkins/plugin/i/built-on-column.svg?color=blue&label=installations[Jenkins Plugin Installs]]
+link:https://plugins.jenkins.io/built-on-column/[image:https://img.shields.io/jenkins/plugin/v/built-on-column.svg[Plugin]]
+
+Adds a column that shows the actual node the last build was run on.
```
Comment (oleg-nenashev) [inline README.adoc:9]: A screenshot would be a good addition if there is no other documentation available on Wiki

Comment (bromps) [reply README.adoc:9]: I agree. I'll provide by today hopefully.

Comment (bromps) [reply README.adoc:9]: @oleg-nenashev I had trouble finding a screenshot anywhere on the Internet, so I took a screenshot of my own Jenkins master.

**Comment by bromps** [comment]
@oleg-nenashev This PR is ready to be merged now.

**Review by oleg-nenashev — APPROVED** [review]
Looks good, thanks for addressing the comments!

**Comment by bromps** [comment]
Is there anything blocking the merge?

**Pull request closed without merging**